instrumentation/vertx-core-3.3.0/src/main/java/com/nr/vertx/instrumentation/VertxCoreUtil.java

```diff
@@ -56,6 +56,7 @@ public static void processResponse(Segment segment, HttpClientResponseImpl resp,
                                                    .uri(uri)
                                                    .procedure(END)
                                                    .inboundHeaders(new InboundWrapper(resp))
+                                                   .status(resp.statusCode(), resp.statusMessage())
```

Comment:
Changes in modules < 4.5.0 are mostly adding the status code and message to incoming responses.